pool/cqrs_command/experiment_command.ml

```diff
@@ -219,126 +224,106 @@ end = struct
   ;;
 end
 
-module AddExperimenter : sig
+module AssignAssistant : sig
   include Common.CommandSig
```

Comment:
```ocaml
  include Common.CommandSig with type t = update_role

  val effects : Id.t -> BaseGuard.Authorizer.effect list
```
as a replacement for the all command signatures

- Additionally, why is this an Id and not Experiment.Id?

Reply:
@mabiede done